### PR TITLE
Fix CI

### PR DESCRIPTION
**Pull request by luckysori**

### CODE REVIEW

File: scripts/generate_serialized_contract_files.sh [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-CONTRACT_TEST_FILES=("Offered" "Accepted" "Confirmed" "Confirmed1" "Signed" "Signed1" "PreClosed" "Closed")
```
Comment (holzeis) [inline scripts/generate_serialized_contract_files.sh:5]: ❓ Why did you remove the Offered, Accepted and Confirmed test cases?

**Review by holzeis — APPROVED** [review]
Thanks 👍 Some tests might now be failing because of the issue that should be fixed with https://github.com/get10101/rust-dlc/pull/11

**Comment by luckysori** [comment]
Not a priority atm, and it was annoying.
